src/test/suite/fixtures/recorded/languages/java/chuckTypeNear.yml

```diff
@@ -0,0 +1,37 @@
+spokenForm: chuck type near
+languageId: java
+command:
+  actionName: remove
+  partialTargets:
+    - type: primitive
+      modifier: {type: containingScope, scopeType: type, includeSiblings: false}
+      mark: {type: decoratedSymbol, symbolColor: default, character: 'n'}
+  extraArgs: []
+initialState:
+  documentContents: |-
+    public class Java {
+        private Java(String name) {
+            String value = "hello";
+        }
+    }
+  selections:
+    - anchor: {line: 3, character: 1}
+      active: {line: 3, character: 1}
+  marks:
+    default.n:
+      start: {line: 1, character: 24}
+      end: {line: 1, character: 28}
+finalState:
+  documentContents: |-
+    public class Java {
+        private Java( name) {
```

Comment:
This one is a bit annoying but probably out of scope for this PR

Reply:
One way we could avoid that would be to include a trailing space as a delimiter. But I don't think spaces are tokens so we going to have to do that text wise without the tree sitter.

Reply:
Yeah I think in the future we could find a way to let token spacing treatment handle this one. Happy to call this out of scope for this PR

Reply:
This problem is now fixed :)